int32 position
float32 speed
int32 position_max
float32 speed_max
float32 acceleration_max
bool enabled